README.md

```diff
@@ -444,21 +465,21 @@ We don't do anything special here, though the following step is more interesting
 @celery_app.task
 def monitor():
     # ...
-    
+
     query = f"""
     INSERT INTO signals(received,url,http_status,available)
     VALUES(systimestamp(), '{signal.url}', {signal.http_status}, {signal.available});
     """
 
-    with engine.connect() as conn:
+    with pool.connection() as conn:
         conn.execute(query)
 ```
 
 Congratulations! You just arrived at the last part of the backend service implementation. We did many things and built a service that can periodically check the website's status, save it in the database, and expose the results through an API.
 
 The very last thing we need to address is to allow connections initiated by the frontend later on. As it will run on localhost:3000 and we don't use domain names, the port is different hence all requests will be rejected with errors related to [Cross-Origin Resource Sharing](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS).
 
-To address this issue, add the following middleware to the application which will allow us to connect at [http://localhost:3000](http://localhost:3000):
+For the sake of simplicity, we are going to allow all origins. This is highly not recommended in production, but for the tutorial, it will be sufficient for us.

```

Comment:
Absolutely make sense! Thank you for the recommendation.

> Also, should we elaborate why it is not recommended, or what kind of stuff should be defined in production?

Actually, the reason it is not recommended originates from attacks that could be performed when the CORS settings are using `*` for origins. Since it is a tutorial, it does not really matter since it won't run in production as is.

Reply:
I am not technical enough to decide but maybe add this as a reminder to part of the sentence? Something like:

"We highly recommended to avoid this in production for security reason but for the tutorial, it will be sufficient for us."

I just find myself asking this question when reading it but I trust your thought on this :)